BDArmory/Control/VesselSpawner.cs

```diff
@@ -996,7 +996,8 @@ private IEnumerator SpawnAllVesselsOnceCoroutine(SpawnConfig spawnConfig)
             {
                 foreach (var vesselName in spawnedVessels.Keys)
                 {
-                    CheckAIWMCounts(spawnedVessels[vesselName].Item1);
+                    // TODO[aubrey]: consider using the new coroutine-based WM check
+                    //CheckAIWMCounts(spawnedVessels[vesselName].Item1);
```

Comment:
Here's the one piece i needed to comment out to get a functional build.

Reply:
What issue was it causing?